src/dao/github/repository.js

```diff
@@ -0,0 +1,29 @@
+const BaseDao = require('~/src/dao').BaseDao
+const Repository = require('~/src/models/github/Repository')
+
+let dao
+
+exports.createDao = function () {
+  dao = new BaseDao({
+    modelType: Repository
+  })
+}
+
+/**
+* Upsert a new Repository record in the database, do nothing on conflict
+* @param id {Number} - github repository id
+* @param name {String} - name of repository
+* @param full_name {String} - owner + repository name e.g. druotic/cool-repo
+* @param installation_id {Number} - foreign key id for associated installation
```

Comment:
Ugghh... not a fan of mixing cases like this. Do you think it's worth normalizing this data in our models? (using fashion model's `coerce`)

Reply:
I think it's worth discussing how we want our db models to look now so that we don't end up having mixed casing.

Reply:
Can you give an example? It seems like `coerce` is meant for type coercion rather than coercing property names, or the docs certainly don't give any examples.

Ultimately, I don't care at all about how fashion model/typing is used, I just want it to be obvious what the casing of a payload looks like without an excessive number of abstractions.

Reply:
Sorry, was thinking of `init` not coerce.

Here's an example using it.
```
const MyModel = require('fashion-model').extend({
  properties: {
    name: String,
    property: String
  },

  init (input) {
    this.setProperty(input.some_property)
  }
})

const model = MyModel.wrap({
  name: 'Jams',
  some_property: 'biebers'
})
```

Adds some extra cruft so not sure if worth or not (might be more performant to just change the data going in before it is passed into the model). I just really don't like having the mixed casing going on.